rust/src/push/base_rules.rs

```diff
@@ -142,7 +142,7 @@ pub const BASE_APPEND_OVERRIDE_RULES: &[PushRule] = &[
         default_enabled: true,
     },
     PushRule {
-        rule_id: Cow::Borrowed("global/override/.m.is_user_mention"),
+        rule_id: Cow::Borrowed("global/override/.m.rule.is_user_mention"),
```

Comment:
Can we check with client teams that they're not also using the wrong names?

Reply:
This seems unsupported in the mobile clients:
- https://github.com/vector-im/element-ios/issues/7567
- https://github.com/vector-im/element-android/issues/8489

Web implementation seems to not yet be updated for the final non-MSC names:
https://github.com/matrix-org/matrix-js-sdk/blob/develop/src/@types/PushRules.ts#L140

Reply:
https://github.com/matrix-org/matrix-js-sdk/pull/3397 looks to use the correct names.